portfolio/src/main/webapp/experience.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Experience</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div class="nav" id="mynav">
+		<a href="index.html">Home</a>
+		<a href="project.html">Projects</a>
+		<a class="active" href="experience.html">Experiences</a>
+		<a href="#" class="icon" onclick="myFunction()">&#9776;</a>
```

Comment:
What is the character? AFAIK HTML supports UTF-8 so you wouldn't need the character constant?

Reply:
This is the trigram for heaven character, the icon with three horizontal lines for the mobile screen.

Reply:
ok